recipe/meta.yaml

```diff
@@ -31,9 +33,12 @@ test:
 
 about:
   home: http://gitorious.org/projects/git-python/
-  license: BSD License
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
Updated to be cleaner, clearer, and compliant. See [license]( https://github.com/gitpython-developers/GitPython/blob/1.0.2/LICENSE ) at this release.